testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -229,12 +230,14 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
       versionOpt: Option[BitcoindVersion] = None): BitcoindInstance = {
     val uri = new URI("http://localhost:" + port)
     val rpcUri = new URI("http://localhost:" + rpcPort)
-    val configFile = writtenConfig(
-      uri,
-      rpcUri,
-      zmqPort,
-      pruneMode,
-      blockFilterIndex = versionOpt.contains(BitcoindVersion.Experimental))
+    val configFile =
+      writtenConfig(uri,
+                    rpcUri,
+                    zmqPort,
+                    pruneMode,
+                    blockFilterIndex = versionOpt.contains(
```

Comment:
maybe move this up a few lines into a intermediate val and say 

```scala
val hasNeutrinoSupport = Vector(BitcoindVersion.Experimental, BitcoindVersion19).contains(version)
.
.
.
blockFilterIndex = hasNeutrinoSupport)
```

Reply:
> is header a hash or a BlockHeader?

It's the header for the block filter 

https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki#filter-headers